app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,13 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
+    @comments = Comment.where(post_id: @user.posts.pluck(:id))
```

Comment:
- [ ] The comments rendered on the post index page should be the ones gotten from the `recent_comments` method in the Post model. Kindly use that.